ui/apps/akasha/jest.setup.js

```diff
@@ -88,3 +94,5 @@ mockIntersectionObserver.mockReturnValue({
 });
 
 global.IntersectionObserver = mockIntersectionObserver;
+
+Object.defineProperty(window, 'localStorage', { value: localStorageMock });
```

Comment:
We should not pollute the global scope, no need to overwrite it, just import the mocked localstorage where you need it in the tests

Reply:
Hmm fair enough, i have moved that line into the feed spec. But the way to mock local storage is to just use: 
`Object.defineProperty(window, 'localStorage', { value: localStorageMock });`

And then that will be applied only in the context of the feed spec.

Reply:
localStorage object can't be directly spied on as long as I am aware. The best you can do is something like
`jest.spyOn(Storage.prototype, 'setItem');` Instead of complicating the code `Object.defineProperty(window, 'localStorage', { value: localStorageMock });` does what needs to be done safely.

Reply:
:) but if the name 'window' is disturbing. I've just changed it to global.